DigitalLearningSolutions.Web/Views/Supervisor/Shared/_TagRow.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@using DigitalLearningSolutions.Data.Models.SelfAssessments;
```

Comment:
_TagRow.cshtml already exists in the solution

Reply:
I think there should be separation of concern regarding the _TagRow.cshtml, the one that exist in the solution is in learning portal not in the supervisor